### ISSUE TITLE
CodeMinion solution misses Python.Runtime.csproj

### ISSUE THREAD

**Issue by epignatelli** [SciSharp/CodeMinion#6]
I can't compile any project from the CodeMinion solution, error below:
<details>

```
Severity	Code	Description	Project	File	Line	Suppression State
Error	NU1104	Unable to find project 'C:\Users\epignatel\Documents\repos\pythonnet\src\runtime\Python.Runtime.csproj'. Check that the project reference is valid and that the project file exists.	Python.Included	C:\Users\epignatel\Documents\repos\codeminion\src\Python.Included\Python.Included.csproj	1	
```

</details>

Specifically, the Python.Runtime project does not load (unavailable).

Am I missing something?

**Comment by epignatelli** [comment]
I solved by cloning the https://github.com/SciSharp/pythonnet repository at the same folder level as the CodeMinion solution.

```
root
|- CodeMinion
|- pythonnet
```

Is it worth adding it to the readme? I can pr a diff if it feels right for you guys.

**Comment by epignatelli** [comment]
Apologies, I am still struggling with compiling this code.
I am compiling the `DebugWin` configuration, running Windows 10 and Visual Studio 2017.
Is there any documentation on which build configuration to use?

Some context:
I am trying to generate a slightly different version of `Numpy`.
While using the configuration above, the code compiles without errors.
Although, I cannot see any console application in `CodeMinion.ApiGenerator/bin/Debug/*/`.
Am I missing something?

**Comment by henon** [comment]
> pythonnet

You are right, this should be documented. By the way, the correct repo to use is: https://github.com/henon/pythonnet_netstandard

**Comment by henon** [comment]
> Apologies, I am still struggling with compiling this code.
> I am compiling the `DebugWin` configuration, running Windows 10 and Visual Studio 2017.
> Is there any documentation on which build configuration to use?
> 
> Some context:
> I am trying to generate a slightly different version of `Numpy`.
> While using the configuration above, the code compiles without errors.
> Although, I cannot see any console application in `CodeMinion.ApiGenerator/bin/Debug/*/`.
> Am I missing something?

Use the `Debug` configuration please. The Target is `.NET Core 2.1`

**Comment by epignatelli** [comment]
> You are right, this should be documented. By the way, the correct repo to use is: https://github.com/henon/pythonnet_netstandard

Precious, thanks very much. I think [git submodules](https://git-scm.com/book/it/v2/Git-Tools-Submodules) might be very useful in this specific case. Do you have any plan on that already?


> Use the `Debug` configuration please. The Target is `.NET Core 2.1`

I'll give it a try, thanks

**Comment by henon** [comment]
I agree. Git submodules would be the perfect solution for CodeMinion, also for the generated libraries. Right now, whenever I release a new version I have to copy the updated generated code from CodeMinion to the separate library repos. It is a shame I didn't I think of it myself ... ;). However, I am quite busy right now, so I'll not be able to change it right now.
